File: C_Programming/Practise_Programs/PP_062.c
/*
==============================================================================
 File Name   : PP_062.c

 Program     : Accept and Display Array Elements Using a Loop

 Description : Accepts five integer elements from the user, stores them in an
               array using a for loop, and displays all the entered elements.

 Author      : Shubham Gadhe

 Time Complexity  : O(n)
 Space Complexity : O(1)
==============================================================================
*/

#include<stdio.h>

int main()
{
    int Arr[5] = {0};

    int iCnt = 0;

    // Accept array elements from the user
    printf("Enter the elements : \n");

    for(iCnt = 0; iCnt < 5; iCnt++)
    {
        scanf("%d", &Arr[iCnt]);
    }

    // Display all elements of the array
    printf("Elements of Array of : \n");

    for(iCnt = 0; iCnt < 5 ; iCnt++)
    {
        printf("%d\n", Arr[iCnt]);
    } 

    return 0;   
}

/*
==============================================================================
 Sample Input
------------------------------------------------------------------------------
10
20
30
40
50

 Sample Output
------------------------------------------------------------------------------
Elements of Array of :
10
20
30
40
50
==============================================================================
*/

File: Java_Programming/Practise_Programs/PP_062.java
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_062
//
//  Description       : This program accepts the number of rows
//                      and columns from the user and displays
//                      '*' in odd rows and '$' in even rows.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 07/08/2026
//
//  Time Complexity   : O(r × c)
//  Space Complexity  : O(1)
//
//////////////////////////////////////////////////////////////////

import java.util.*;

class PP_062
{
    //////////////////////////////////////////////////////////////////
    //
    //  Function Name    : Display()
    //
    //  Description      : It is used to display '*' in odd
    //                     numbered rows and '$' in even
    //                     numbered rows for the specified
    //                     number of columns.
    //
    //  Author           : Shubham Somanath Gadhe
    //  Date             : 07/08/2026
    //
    //////////////////////////////////////////////////////////////////

    public static void Display(int iRow, int iCol)
    {
        int i = 0, j = 0;

        for(i = 1; i <= iRow; i++)
        {
            for(j = 1; j <= iCol; j++)
            {
                if(i % 2 == 0)
                {
                    System.out.print("$\t");
                }
                else
                {
                    System.out.print("*\t");
                }
            }

            System.out.println();
        }
    }

    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue1 = 0, iValue2 = 0;

        System.out.println("Enter the number of Rows: ");
        iValue1 = sobj.nextInt();

        System.out.println("Enter the number of Columns : ");
        iValue2 = sobj.nextInt();

        Display(iValue1, iValue2);
    }
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input  :
//      Enter the number of Rows :
//      4
//
//      Enter the number of Columns :
//      4
//
//  Sample Output :
//
//      *    *    *    *
//      $    $    $    $
//      *    *    *    *
//      $    $    $    $
//
//////////////////////////////////////////////////////////////////